test/unit_test/workflows/mock_data/readymade_system.json

```diff
@@ -0,0 +1,120 @@
+{
+    "azureml_client": {
+        "subscription_id": "my_subscription_id",
+        "resource_group": "my_resource_group",
+        "workspace_name": "my_workspace"
+    },
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "model_path": "models/resnet_trained_for_cifar10.pt",
+            "io_config": {
+                "input_names": ["input"],
+                "input_shapes": [[1, 3, 32, 32]],
+                "output_names": ["output"],
+                "dynamic_axes": {
+                    "input": {"0": "batch_size"},
+                    "output": {"0": "batch_size"}
+                }
+            }
+        }
+    },
+    "systems": {
+        "azureml_system": {
+            "type": "AzureNDV2System",
+            "config": {
+                "aml_compute": "cpu-cluster",
```

Comment:
As `AzureNDV2System` has the GPU. How about to change the `cpu-cluster` to `gpu-cluster` to avoid confusion?